docs/How-to-Revendor-a-Dependency.md

```diff
@@ -0,0 +1,46 @@
+# How to Revendor a Dependency
+
+At the time of writing, Homebrew vendors several dependencies within the
+[Library/Homebrew/vendor](https://github.com/Homebrew/brew/tree/master/Library/Homebrew/vendor)
+subdirectory.
+
+This page serves to document the process for revendoring these dependencies.
+
+## Copying the files
+
+Each directory in `vendor/` contains the **contents** of the `lib/` directory for the corresponding
+dependency. Thus, `require`ing a vendored dependency looks like this:
+
+```ruby
+require "vendor/library/file"
+```
+
+Thus, the copying process is as simple as removing the old files, and copying the new files into
```

Comment:
> […] process is almost as simple as […]

(Rationale: the old files need to be removed from the Git index, too.)

Reply:
second "thus" in a row is two thuses too many